src.md

```
@@ -1,61 +1 @@
-
-    // Logika dekripsi untuk 'repliedTo' tetap sama...
-    // ...
-
-    return decryptedMsg;
-  } catch (e) {
-    // ...
-  }
-}
-```
-Perubahan ini membuat `decryptMessageObject` lebih pintar: ia tahu bahwa untuk pesan file, yang perlu didekripsi adalah `fileKey`, bukan `content`. Ini debería menghentikan kebingungan dan panggilan dekripsi yang salah.
+ Security Check Recommended (CVE-2025-55182): Please review your application's dependencies. If you are running React or Next.js applications, immediately update to the latest stable versions (React 19.2.1 or the latest version of Next.js: 15.0.5, 15.1.9, 15.2.6,. 15.3.6, 15.4.8, 15.5.7, 15.6.0-canary.58 or 16.0.7), and republish. https://firebase.google.com/docs/studio/troubleshooting.md#update-app 
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)